optimizer_wrapper.rb

```diff
@@ -350,9 +339,31 @@ def self.split_independent_vrp_by_skills(vrp)
       }
     }
 
+    vrp.relations.select{ |relation|
+      Models::Relation::ON_VEHICLES_TYPES.include?(relation.type)
+    }.each{ |relation|
+      v_skills_indices = relation.linked_vehicle_ids.map{ |v_id| correspondance_hash[v_id] }
+      mission_skill_indices = []
+      next unless v_skills_indices.any?{ |v_index|
+        mission_skills.each_index.any?{ |m_index|
+          mission_skill_indices << m_index if compatibility_table[m_index][v_index]
```

Comment:
It seems dangerous to me to insert elements in this array in a "next unless" condition. In fact I do not quite understand what we want to check here.